.github/workflows/pull_request.yml

```diff
@@ -4,24 +4,22 @@ on:
   push:
     branches:
       - main
+      - 7.x
       - 7.[0-9]+.x
-    paths-ignore:
-      - 'optaplanner-8-benchmarks/**'
 
   pull_request:
     branches:
       - main
+      - 7.x
```

Comment:
Based on Marian's comment:
https://github.com/kiegroup/kie-benchmarks/pull/168#discussion_r850195075

We would like different GH actions for main and for 7.x, am I right? In main we would like to run PR check for main and possibly for 8.[0-9]+.x. Will there be forked branches on release as in 7.x? For example 7.67.x.

Reply:
Thanks @MichalCervenansky , I've just sent another commit.